alloccli/comment.py

```diff
@@ -158,7 +158,7 @@ def run(self, command_list):
         comma = ""
         text = ""
         if recipients:
-            for k_, info in recipients.items():
+            for k_, info in list(recipients.items()):
```

Comment:
Wrapping "list()" around everything feels sort of not so nice to me. I think maybe the 2to3 tool is being overzealous? Maybe shake the trees and see if that's strictly necessary?

Reply:
yeah, I'm going to go through each of these.

Reply:
additional info: https://docs.python.org/3/whatsnew/3.0.html#views-and-iterators-instead-of-lists

I'm still investigating it xD

Reply:
right, I seems that 2to3 was indeed overly careful. I've never heard of a 'view' before, but according to [this page](https://docs.python.org/3/library/stdtypes.html#dict-views) it is fine to loop over them with a for loop. So I guess a 'view' is similar to a list but not the same? either way I'm going to revert all the cases where `list()` was added in for loops.